### ISSUE TITLE
Bug: "Prefer lazy Logger calls." warning while not using a function

### ISSUE THREAD

**Issue by KamilLelonek** [rrrene/credo#375]
### Environment

* Credo version (`mix credo -v`): `0.7.4`
* Erlang/Elixir version (`elixir -v`): `Elixir 1.4.2`
* Operating system: OS X Sierra

### What were you trying to do?

        Logger.info "Successful storing for cursor: #{id} and epoch: #{epoch} from stream: #{stream}."
        Logger.info "The Changes were #{inspect changes}."

### Expected outcome

    179 mods/funs, found no issues.

### Actual outcome

      Warnings - please take a look
    ┃
    ┃ [W] ↗ Prefer lazy Logger calls.
    ┃       lib/ds_postgres_importer/monitor/monitor.ex:40 #(Monitor.log)
    ┃ [W] ↗ Prefer lazy Logger calls.
    ┃       lib/ds_postgres_importer/monitor/monitor.ex:39 #(Monitor.log)

**Comment by rrrene** [comment]
This is the expected behaviour of the check. Why do you consider this a bug?

There is a config parameter if you don't want to get these for `:info` calls. For example, you can use

```elixir
{Credo.Check.Warning.LazyLogging, ignore: [:error, :warn, :info]}
```

to only report `Logger.debug` calls.

**Comment by KamilLelonek** [comment]
Why lazy logger calls all preferred in case of just interpolating variables (not functions)?

**Comment by rrrene** [comment]
It was discussed when introducing the check: https://github.com/rrrene/credo/pull/326#issuecomment-285310106

But you are raising a point with your example code: maybe we should just warn for `debug` calls.

**Comment by rrrene** [comment]
Credo `v0.8.0-rc3` does only warn for `debug` calls by default. Thx again! :+1:
